specification/hybridcompute/resource-manager/readme.ruby.md

```diff
@@ -0,0 +1,15 @@
+## Ruby HybridCompute
+
+These settings apply only when `--ruby` is specified on the command line.
+
+``` yaml
+package-name: azure_mgmt_hybridcompute
+package-version: "0.17.1"
```

Comment:
Any specific reason why we're setting the version as .17.1?

Reply:
oops bad copy paste